app/lib/finance/billing.rb

```diff
@@ -14,6 +14,10 @@ def create_line_item(params)
       li = LineItem.new(params, {without_protection: true})
       li.errors.add(:base, :invalid_invoice_state)
       li
+    rescue ActiveRecord::SubclassNotFound
+      li = LineItem.new(params.except(:type), {without_protection: true})
+      li.errors.add(:base, :invalid_invoice_state)
```

Comment:
```suggestion
      li.errors.add(:base, : invalid_line_item_type)
```
?

Reply:
Not really it is the same as above just removing the `type` in params